### PR TITLE
Fedora 35

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: repos.d/rpm/fedora.yaml [hunk 1]
```diff
@@ -386,6 +386,47 @@
...
+    - desc: Fedora repository for package maintenance
+      url: https://src.fedoraproject.org/
+  packagelinks:
+    - type: PACKAGE_SOURCES
+      url: 'https://src.fedoraproject.org/rpms/{srcname}/tree/f35'
+    - type: PACKAGE_RECIPE
+      url: 'https://src.fedoraproject.org/rpms/{srcname}/blob/f35/f/{srcname}.spec'
+    - type: PACKAGE_RECIPE_RAW
+      url: 'https://src.fedoraproject.org/rpms/{srcname}/raw/f35/f/{srcname}.spec'
+  tags: [ all, fedora, rpm ]
```
Comment (AMDmi3) [inline repos.d/rpm/fedora.yaml:428]: Please add `production` tag here for it to appear on main Repology instance

File: repos.d/rpm/fedora.yaml [hunk 2]
```diff
@@ -386,6 +386,47 @@
...
   tags: [ all, production, fedora, rpm ]
 
+- name: fedora_35
+  type: repository
+  desc: Fedora 35
+  statsgroup: Fedora
+  family: fedora
+  ruleset: [fedora, rpm]
+  color: '294172'
+  minpackages: 25000
```
Comment (AMDmi3) [inline repos.d/rpm/fedora.yaml:396]: This fails as it only has 22581 packages. You should probably leave 22000 everywhere, is it's the fair number of packages without duplicates.

**Review by AMDmi3 — CHANGES_REQUESTED** [review]

File: repos.d/rpm/fedora.yaml (hunk 2 above)
Comment (keszybz) [reply repos.d/rpm/fedora.yaml:396]: Oh, this is about source packages… Yeah, I think we have 23.5k now.

**Review by AMDmi3 — APPROVED** [review]

**Comment by AMDmi3** [comment]
Thanks you!

**Pull request merged**